src/createViewer.js

```diff
@@ -116,27 +114,17 @@ const createViewer = (
 
         store.itkVtkView.setClickCallback(lastPickedValues => {
           if (lastPickedValues.value !== null) {
-            store.imageUI.selectedLabel =
-              lastPickedValues.value[lastPickedValues.value.length - 1]
-
-            if (
-              store.imageUI.lastSelectedLabel === store.imageUI.selectedLabel
-            ) {
-              if (store.imageUI.selectedLabel !== 'all') {
-                const currentWeight =
-                  store.imageUI.labelMapWeights[store.imageUI.selectedLabel]
-                if (currentWeight > 0.5) {
-                  store.imageUI.labelMapWeights[
-                    store.imageUI.selectedLabel
-                  ] = 0.0
-                } else {
-                  store.imageUI.labelMapWeights[
-                    store.imageUI.selectedLabel
-                  ] = 1.0
-                }
```

Comment:
I preferred to see the weight of each label when clicking it, instead of immediately changing it to 1.  But this is behavior, together with the meaning of the `All` selection, is really up to what you prefer.

Reply:
Makes sense. This is a refinement to move towards a preference of efficient interaction, i.e. just one click.